simphony/scripts/material_realtion_generate.py

```diff
@@ -0,0 +1,244 @@
+import click
+import yaml
+from simphony.core.keywords import KEYWORDS
+
+
+def generate_class_import():
+    return [
+        "from simphony.material_relations.material_relation import (\n",
+        "\tMaterialRelation)\n",
+        "from simphony.core.cuba import CUBA\n",
+        "import simphony.core.data_container as dc\n",
+        "\n",
+        "\n",
+    ]
+
+
+def generate_class_header(mr):
+    return [
+        "class {MR_NAME}(MaterialRelation):\n".format(
+            MR_NAME=mr['class_name']
+        ),
+        "\n"
+    ]
+
+
+def generate_description_block(mr):
+    return [
+        "\t\"\"\" Automatically generated implementation of the\n",
```

Comment:
Maybe stick the 'Automatically generated...' phrase not in the class's docstr but instead use it as a comment at the top generated python file.